README.md

```diff
@@ -225,6 +225,16 @@ env | grep TEST
 # TEST=value#notcomment
 ```
 
+However, unlike bash (but like fish), the `#` cannot be at the start, so as to not confuse with comments.
```

Comment:
Thanks for merging my updated documentation, however, I intend to remove this whole section in my PR. I decided it is overly verbose.

Reply:
Then I will update it again later ;)

Reply:
I tried to make it less verbose, but I think it is worth to be mentioned.

Awaiting your response ;)

Reply:
I close this, if you are interested in changing the documentation completely at this point we should move it to an own Issue imho.